services/verification/src/utils.ts

```diff
@@ -251,7 +257,7 @@ function validateSolcPath(solcPath: string, log: InfoErrorLogger): boolean {
 async function getSolcExecutable(
   version: string,
   log: InfoErrorLogger
-): Promise<string> {
+): Promise<string | null> {
```

Comment:
Isn't it better here to handle with an undefined or throwing an error instead of `string | null`